### ISSUE TITLE
Architectures other than x86 and ARM are no longer supported

### ISSUE THREAD

**Issue by Shnatsel** [rustsec/rustsec#962]
With the migration to rustls, we are running into the limitation of its cryptographic library, `ring`, only supporting x86 and ARM targets. The issue is tracked upstream in ring: https://github.com/briansmith/ring/issues/1455

The generic fallback implementations written in C were added for most primitives in git master, but were never released. The maintainer has not touched the project in 8+ months, so it's not clear if/when the support will materialize.

**Comment by Shnatsel** [comment]
`ring` 0.17 has been released with architecture-independent fallback implementations. We're waiting on `rustls` to incorporate it: https://github.com/rustls/rustls/pull/1508

**Comment by Shnatsel** [comment]
The change has been merged into master, but we are still waiting on it being backported to the current series: https://github.com/rustls/rustls/pull/1525

**Comment by Shnatsel** [comment]
Upstream `rustls` has shipped a point release that updates to a newer version of `ring`. Now `cargo audit` v0.18.3 can be compiled for obscure CPU platforms without issue.
